spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/ApiDocumentation.java

```diff
@@ -124,6 +124,7 @@ public void index() throws Exception {
 				linkWithRel("tasks/platforms").description("Provides platform accounts for launching tasks.  The results can be filtered to show the platforms that support scheduling by adding a request parameter of 'schedulesEnabled=true"),
 				linkWithRel("tasks/logs").description("Retrieve the task application log"),
 				linkWithRel("tasks/thinexecutions").description("Returns thin Task executions"),
+				linkWithRel("tasks/thinexecutions/name").description("Returns all thin Task executions for a given Task name"),
```

Comment:
I noticed in the source being ported there was a removal of `linkWithRel("tasks/thinexecutions")` ([here](https://github.com/spring-cloud/spring-cloud-dataflow/pull/5994/files#diff-9158f2aa24bd9b317120f16404df0a58043b47e3479e147777f71dba6e36cba8L125)). Is this needed here as well?

Reply:
There was a duplicate